test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -144,8 +147,143 @@ def forward(self, *x):
         onnx_program = torch.onnx.export(VisionModel(), args, dynamo=True)
         onnx_testing.assert_onnx_program(onnx_program)
 
+    def test_onnx_export_huggingface_llm_models_with_kv_cache(self):
```

Comment:
This file is meant to house end to end tests for "small" models whose definition being just a few ops. A HF model should be placed elsewhere. Consider creating a file just for testing HF models to make change management easier.

Reply:
Done

Reply:
Created test_hf_models_e2e.py for llm testing.